app/src/main/java/utils/Convertion.java

```diff
@@ -0,0 +1,49 @@
+package utils;
+
+/**
+ * Created by remi on 02/11/2016.
+ */
+
+public class Convertion {
+
+    /**
+     * Convert the time in string
+     * @param time time in hours
+     * @return the string represent the time
+     */
+    public static String timeInString(double time){
```

Comment:
Can't you use the method "dateHourMinuteToString(Date date)" in the class "DateManipulation" instead of re-create your own method ?